frontend/viewer/src/lib/components/lcm-rich-text-editor/lcm-rich-text-editor.svelte

```diff
@@ -15,6 +20,7 @@
       span: {
         selectable: false,
         content: 'text*',
+        inline: true,
```

Comment:
funny I think I removed this last time to fix some other issues lol.

Reply:
Yeah...I'm not entirely sure if I needed this. But...I've played with it quite a bit and didn't see any errors.
But...I can't say I'd be too surprised if it's still possible to trigger an error 😜.

Reply:
Nice catch. The issue was that a trailing \n doesn't seem to get rendered.
Removing `inline: true,` did fix it in some cases (behaviour was different depending on whether I removed it from doc or span), but then I ran into worse bugs (e.g. backspace and delete removing entire nodes).

So, I fixed it by always including a zero-width-space after a \n to ensure it always gets rendered.

Reply:
Actually, I didn't end up pushing that fix, because it has the obvious caveat that there are then 2 characters, so you need to e.g. backspace twice. I wasn't able to find a good way to fix this. It's like prose-mirror wants us to use `inline: false`, but...that creates worse issues (which I've now [added comments](https://github.com/sillsdev/languageforge-lexbox/pull/1802/commits/fd5583580c0cfa4877d3b514d4f61e369e38f00c) about).

I played with always added a \n at the end of the last span so the next user-added \n actually does something...That sort of worked, but is weird and would have required a bunch of test changes, because playwright was inserting text after that \n.

Yeah...I'm not sure what's best, but...I'd be ok with ignoring it for now.

Reply:
Actually, I fixed it in [0de2595](https://github.com/sillsdev/languageforge-lexbox/pull/1802/commits/0de25952af42ea76c56d28e2e3ba32b9188ab0bb).
I copied what PoseMirror does when inline !== true. It seems that <br> is magically the right way to get a trailing \n to render.